components/pipeline-service/development/update-tekton-config-performance.yaml

```diff
@@ -43,3 +43,6 @@
   path: /spec/platforms/openshift/pipelinesAsCode/options/deployments/pipelines-as-code-webhook/spec/replicas
   # default pipeline-service setting is 1
   value: 2
+- op: replace
+  path: /spec/pipeline/options/horizontalPodAutoscalers/tekton-pipelines-webhook/spec/minReplicas
+  value: 6
```

Comment:
do we really want to bump this for dev overlay?

Reply:
I wanted to vet that the setting worked with e2e's @hugares 

let's get a clean e2e, then I'll undo it.

will put a hold on in the interim